framework/src/modules/network/index.js

```diff
@@ -49,12 +49,30 @@ module.exports = class NetworkModule extends BaseModule {
 
 	get actions() {
```

Comment:
Same as before

Reply:
Not sure which ones should be public here?
Made everything public except for `getPeersCountByFilter`.
@shuse2 @lsilvs

Reply:
everything can be false i think, it will not be called from peers directly.
Also, `emitisPublic`?

Reply:
Removed this as it was unused. Waiting for _greenies_ (y)